### ISSUE TITLE
with v1.5.11: HmIP communiation = OK, but Hm communication = broken?, 

### ISSUE THREAD

**Issue by juersei** [ioBroker/ioBroker.hm-rpc#510]
After yesterday´s update to 1.5.11 all data of my HMiP sensors are again flawlwssly recorded in my SQL database, very nice!

However, at the same time it seems that the HM sensors stopped to communicate, whereas I see this data without any interruption recorded in my CCU2.

Strange thing is that with 1.5.10 (1.5.8, 1.5.9) it was just the opposite in my case, HmIP comm. = broken, HM comm. = OK.

Anyone else making the same observtion?

**Comment by foxriver76** [comment]
Then you should check the adapter ports and if they are accessible by your. CCU, probably Adapter port of both instances has switched.

**Comment by juersei** [comment]
I confirm that **after a reboot of ioBroker** the communication of all instances was fixed and everything is now working fine with 1.5.11 since about 15 hours.
 
The problems were indeed related to the port mapping of the homematic ports (2001/2010) to the ioBroker ports (2001,2002 .../ 2010, 2011, ...) which is a little more tricky in my case with two CCUs at two different locations (connected via a permanent VPN. I will write a summary of the findings later before I close this thread. 
These issues happened  due to several updates incl. node and jscontroller where I did not change any setting (e.g. ports) myself.
  
I have not touched v1.5.12 yet, I will do it after I see my current installation being stable and reliable.

Thanks for the very good hints which I received to debug my problems.
